src/poliastro/plotting/porkchop.py

```diff
@@ -23,6 +20,172 @@
 from poliastro.util import norm
 
 
+class PorkchopPlotter:
+    def __init__(
+        self,
+        departure_body,
+        target_body,
+        launch_span,
+        arrival_span,
+        ax=None,
+        tfl=True,
+        vhp=True,
+        max_c3=45.0 * u.km ** 2 / u.s ** 2,
+        max_vhp=5 * u.km / u.s,
+    ):
+        """
```

Comment:
Move this docstring to the class (instead of `__init__`) and add one line summary before `Parameters`

Reply:
Done!